pkg/pipelines/bootstrap.go

```diff
@@ -290,3 +295,12 @@ func defaultPipelines(r scm.Repository) *config.Pipelines {
 		},
 	}
 }
+
+// Checks whether the pipelines.yaml is present in the output path specified.
+func checkPipelinesFileExists(appFs afero.Fs, outputPath string, overWrite bool) error {
+	exists, _ := ioutils.IsExisting(appFs, filepath.Join(outputPath, pipelinesFile))
+	if exists && !overWrite {
+		return fmt.Errorf("Pipelines.yaml in output path already exists.To overwrite, please set --overwrite flag and re-run command")
```

Comment:
How about a space between `exists.` and `To` ?

How about "If you want replace your existing files, please rerun with --overwrite."

Reply:
made the required change